awx/main/models/events.py

```diff
@@ -58,6 +59,11 @@ def create_host_status_counts(event_data):
 
 
 def emit_event_detail(event):
+    if (
+        settings.UI_LIVE_UPDATES_ENABLED is False and
+        event.event not in ('playbook_on_stats', 'EOF')
```

Comment:
I'd like to maybe make this be `not startswith("runner_")`, but I'm having some trouble getting the UI to behave in a reasonable way cc @jakemcdermott is taking a look, I think.